config/cli.go

```diff
@@ -128,6 +137,11 @@ func ReadCLIConfig(ctx *cli.Context) (*CLIConfig, error) {
 		LogsDirectory: ctx.String(LogsDirectoryFlagName),
 	}
 
+	if ctx.Bool(DocsFlagName) {
+		printDocLinks()
+		os.Exit(0)
+	}
+
```

Comment:
instead of exiting here, it would be better to return the flag inside the cfg, then cancel in main.go#SupersimMain using `closeApp` if the flag is set. That aligns better with using `closeApp` as the main way to shutdown.

Reply:
if u end up making this a command, you can check in main `ctx.Command` before running `closeApp`

Reply:
Done!